cli/pkg/validate/required_keys.go

```diff
@@ -46,6 +46,14 @@ func (rk *RequiredKeysValidator) Validate(dc *catalog.DataCatalog) []ValidationE
 				})
 			}
 
+			// Validate property name doesn't have leading or trailing whitespace
+			if prop.Name != "" && prop.Name != strings.TrimSpace(prop.Name) {
```

Comment:
```suggestion
	if prop.Name != strings.TrimSpace(prop.Name) {
```
Should this check be enough ?